packages/web/src/components/collection/CollectionCard.test.tsx

```diff
@@ -89,12 +93,63 @@ describe('CollectionCard', () => {
     ).toBeInTheDocument()
   })
 
-  it('shows the number of reposts and favorites with the correct screen-reader text', () => {
-    renderCollectionCard()
-    expect(screen.getByTitle('Reposts')).toBeInTheDocument()
-    expect(screen.getByText('10')).toBeInTheDocument()
+  it('hidden collections are rendered correctly', () => {
+    renderCollectionCard({
+      reduxState: {
+        collections: { entries: { 1: { metadata: { is_private: true } } } }
+      }
+    })
+
+    expect(
+      screen.getByRole('button', { name: /test collection test user hidden/i })
+    ).toBeInTheDocument()
+  })
+
+  it('premium locked collections are rendered correctly', () => {
+    renderCollectionCard({
+      reduxState: {
+        collections: {
+          entries: {
+            1: {
+              metadata: {
+                access: { stream: false },
+                stream_conditions: {
+                  usdc_purchase: { price: 10, albumTrackPrice: 1, splits: {} }
+                }
+              }
+            }
+          }
+        }
+      }
+    })
```

Comment:
lol ok maybe this one is slightly less clean, but not your fault

Reply:
yeah maybe we consider the {users: [{...}], tracks: [{...}]} pattern soon